src/components/Info.css

```diff
@@ -0,0 +1,51 @@
+.Info-icon {
+  height: 17px;
+  width: 17px;
+  margin-left: 10px;
+  margin-top: 7px;
+}
+
+.Info-icon:hover {
+  cursor: pointer;
+}
+
+.Info-header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.Info-header h2 {
+  color: #fff;
+}
+
+.Text-container {
+  position: relative;
+}
+
+.Show {
```

Comment:
This isn't a good/descriptive class name. If `show` is used, it's typically in combination with another class.

Reply:
Done